h2o-algos/src/main/java/water/rapids/prims/mungers/mungers/AstTargetEncoderTransform.java

```diff
@@ -121,14 +121,14 @@ private Frame getFrameToTransform(Env env, Env.StackHelp stk, AstRoot asts[]) {
   private byte getDataLeakageHandlingStrategy(Env env, Env.StackHelp stk, AstRoot asts[]) {
```

Comment:
I thik this should return DataLeakageHandlingStrategy object, working with the byt val directly is dangerous

Reply:
Done, thanks.